openhgnn/dataset/NodeClassificationDataset.py

```diff
@@ -13,6 +15,7 @@
 from . import AcademicDataset, HGBDataset, OHGBDataset
 from .utils import sparse_mx_to_torch_sparse_tensor
 from ..utils import add_reverse_edges
+from torch_sparse import SparseTensor
```

Comment:
Where did you need this? And we are not allow to use functions in `pytorch_geometric`.